### DISCUSSION TITLE
How would work when having multiple owners of one DependentResource?

### DISCUSSION THREAD

**Discussion by juangon** [operator-framework/java-operator-sdk#1624] [Q&A] [answered]
Hi again!

So, I have some kind of "shared" dependent resource. In my case, those are KafkaTopics which of course depends on multiple sides (microservices). Each of those microservices needs the same topics to exists in order to startup properly, so it seems that all of those Custom Resources will need the same DependentResource.

AFAIK JOSDK will add the needed fields (resourceOwner, id,..) to those shared resources in order to know which one is the owner. But in this case there could be multiple owners of the same resource. How that would work? Is the first owner that created it the one who stays? The last one?

Thanks!

**Comment by csviri** [comment, marked as answer]
Hi @juangon ,

The current setup is not really designed for this use case, probably just one will add the owner reference (when the dependent is created). To cover this differently pls create an issue.
